src/services/csv-processor.ts

```diff
@@ -73,18 +78,43 @@ export async function extractTableInformation(fileBuffer: Buffer, fileType: File
       `Unknown error occurred, please refer to the log for more information`,
       FileValidationErrorType.unknown
     );
+  }
+
+  if (type === 'data_table')
```

Comment:
Can we have some braces here please... my brain can deal with no braces if it's a one-liner but not for a 10 line block 😆

Reply:
Braces added